Include/internal/pycore_freelist.h

```diff
@@ -97,6 +109,7 @@ typedef struct _Py_freelist_state {
     struct _Py_float_state float_state;
     struct _Py_tuple_state tuple_state;
     struct _Py_list_state list_state;
+    struct _Py_dict_state dict_state;
```

Comment:
Keeping "state" in the name is misleading.  Let's use a simpler name:

```suggestion
    struct _Py_dict_state dicts;
```

Reply:
We decided to unify schema from https://github.com/python/cpython/pull/113919,
Do you want to revert all changes?

Reply:
I think that would help.  Looks like you're addressing this in #114581.  Thanks!